primitives/keystore/Cargo.toml

```diff
@@ -7,7 +7,6 @@ license = "Apache-2.0"
 homepage = "https://substrate.dev"
 repository = "https://github.com/paritytech/substrate/"
 description = "Keystore primitives."
-documentation = "https://docs.rs/sp-core"
```

Comment:
why did you remove the doc ?

Reply:
It's a mistake, isn't it?

Reply:
documentation is not a valid cargo keyword ?
I don't think it is a mistake, the link is good.

Reply:
ok, thank you